test/integration/test_transmit_worker_process.py

```diff
@@ -168,6 +168,54 @@ def process_method(results_sockfile_read):
 
         self.check_artifacts(str(process_dir), job_type)
 
+    @pytest.mark.parametrize("process_isolation_executable", ['exist', 'does_not_exist'])
+    def test_process_isolation_executable_transmit(self, tmp_path, process_isolation_executable, mocker):
+        """
+        Case transmit should not fail if process isolation executable does not exist
+        """
+        def mock_check_isolation_executable_installed(executable):
+            if executable == "exist":
+                return True
+
+        mocker.patch.object(ansible_runner.interface, 'check_isolation_executable_installed', mock_check_isolation_executable_installed)
+
+        job_kwargs = self.get_job_kwargs('run')
+        job_kwargs['process_isolation'] = True
+        job_kwargs['process_isolation_executable'] = process_isolation_executable
+
+        outgoing_buffer_file = tmp_path / 'buffer_out'
+        outgoing_buffer_file.touch()
+        outgoing_buffer = outgoing_buffer_file.open('b+r')
+
+        transmitter = ansible_runner.interface.run(
+            streamer='transmit',
+            _output=outgoing_buffer,
+            **job_kwargs,
+        )
+
+        # valide process_isolation kwargs are passed to transmitter
+        assert transmitter.kwargs['process_isolation'] == job_kwargs['process_isolation']
+        assert transmitter.kwargs['process_isolation_executable'] == job_kwargs['process_isolation_executable']
+
+        # validate that transmit did not fail due to missing process isolation executable
+        assert transmitter.rc in (None, 0)
```

Comment:
Is `None` really a valid value here?

Reply:
https://github.com/ansible/ansible-runner/blob/devel/test/integration/test_transmit_worker_process.py#L74-L75